packages/v4/src/content/design-guidelines/styles/icons/icons.md

```diff
@@ -110,6 +110,14 @@ Use this for 'fa' regular icons: `<i className="far [insert-icon-name]"></i>`<br
 If you're looking to copy React for an icon:<br />
 Use this for 'pficon' icons: `import { [insert-icon-name] } from '@patternfly/react-icons'`
 
+## Font Awesome solid (FAS) vs Font Awesome regular (FAR)
+The PF icon set supports use of any FAS (font awesome solid) icons. Therefore, designers may use any FAS icon in their designs. If an FAR icon is needed, developers must include the FAR icon(s) themselves. There are a few ways of doing so:
```

Comment:
The tricky part here is that this guidance is for users using the FA webfont that ships with PF core. If they're using `@patternfly/react-icons`, they get any icon that has the icon name under the "React" table column, eg, `AngleDoubleLeftIcon`, so they don't need to do anything special to use any of those icons.

I guess we could start this paragraph with

> The Font Awesome icons included with PatternFly core are FAS (Font Awesome solid) webfont icons. If you're using icons via `<i class="[fa, far, fas] [insert-icon-name]"></i>`, designers and developers may use any FAS icon in their designs. If an FAR icon is needed, developers must include the FAR icon(s) themselves. There are a few ways of doing so: